src/backend/functions/change-requests-new.ts

```diff
@@ -105,105 +117,88 @@ const createStageGateChangeRequest = async (
 };
 
 // Create proper type of new change request
-export const baseHandler: Handler = async ({ body }, _context) => {
+export const baseHandler: Handler<FromSchema<typeof inputSchema>> = async ({ body }, _context) => {
   const { submitterId, wbsElementId, type } = body;
   if (type === CR_Type.DEFINITION_CHANGE || type === CR_Type.ISSUE || type === CR_Type.OTHER) {
-    return createStandardChangeRequest(submitterId, wbsElementId, type, body);
+    return createStandardChangeRequest(
+      submitterId,
+      wbsElementId,
+      type,
+      body as FromSchema<typeof standardSchema>
+    );
   }
   if (type === CR_Type.ACTIVATION) {
     // TODO: is there a better way to convert this date string?
     // I couldn't seem to figure out if middy can handle this, but this 1 additional line isn't the worst
-    body.startDate = new Date(body.startDate);
-    return createActivationChangeRequest(submitterId, wbsElementId, type, body);
+    body = body as FromSchema<typeof activationSchema>;
+    return createActivationChangeRequest(submitterId, wbsElementId, type, {
+      ...body,
+      startDate: new Date(body.startDate)
+    });
   }
   if (type === CR_Type.STAGE_GATE) {
-    return createStageGateChangeRequest(submitterId, wbsElementId, type, body);
+    return createStageGateChangeRequest(
+      submitterId,
+      wbsElementId,
+      type,
+      body as FromSchema<typeof stageGateSchema>
+    );
   }
   // TODO: change this return statement
   return buildClientFailureResponse('CR type not supported');
 };
 
+// standard / scope change request fields
+const standardSchema = bodySchema({
+  type: enumType(CR_Type.OTHER, CR_Type.ISSUE, CR_Type.DEFINITION_CHANGE),
+  submitterId: intType,
+  wbsElementId: intType,
+  what: stringType,
+  scopeImpact: stringType,
+  timelineImpact: intType,
+  budgetImpact: intType,
+  why: arrayType(
+    bodySchema({
+      explain: stringType,
+      type: enumType(
```

Comment:
is there any way we can have this be simplified to `Object.values(Scope_CR_Why_Type)` or is that too much work? the spread operator might help? kinda just guessing and wondering here

Reply:
Yup, that works great since prisma has enums as a union type and object with key value pairs.